nextcord/application_command.py

```diff
@@ -2624,6 +2636,13 @@ def __init__(
             Permission(s) required to use the command. Inputting ``8`` or ``Permissions(administrator=True)`` for
             example will only allow Administrators to use the command. If set to 0, nobody will be able to use it by
             default. Server owners CAN override the permission requirements.
+        nsfw: :class:`bool`
+            If the command should be usable in NSFW channels or not. Setting to ``True`` will disable the command from
```

Comment:
This seems confusing. The name of the parameter is `nsfw`, so I would expect it to enable support in nsfw channels. But it doesn't?

Reply:
Yeah, all of the descriptions need a rework...